### ISSUE TITLE
Sprockets issue

### ISSUE THREAD

**Issue by pyromaniac** [widefix/actual_db_schema#89]
Hey @ka8725, new UI is a great feature but unfortunately is causes issues for the projects not having sprockets in the bundle:

```bash
$ rake db:migrate
rake aborted!
NoMethodError: undefined method `assets' for #<Rails::Application::Configuration:0x0000000000c698> (NoMethodError)
Did you mean?  asset_host
/Users/pyromaniac/.rvm/gems/ruby-3.2.2/gems/railties-7.1.3.4/lib/rails/railtie/configuration.rb:109:in `method_missing'
/Users/pyromaniac/.rvm/gems/ruby-3.2.2/gems/actual_db_schema-0.7.6/lib/actual_db_schema/engine.rb:14:in `block in <class:Engine>'
/Users/pyromaniac/.rvm/gems/ruby-3.2.2/gems/railties-7.1.3.4/lib/rails/initializable.rb:32:in `instance_exec'
/Users/pyromaniac/.rvm/gems/ruby-3.2.2/gems/railties-7.1.3.4/lib/rails/initializable.rb:32:in `run'
/Users/pyromaniac/.rvm/gems/ruby-3.2.2/gems/railties-7.1.3.4/lib/rails/initializable.rb:61:in `block in run_initializers'
/Users/pyromaniac/.rvm/gems/ruby-3.2.2/gems/railties-7.1.3.4/lib/rails/initializable.rb:60:in `run_initializers'
/Users/pyromaniac/.rvm/gems/ruby-3.2.2/gems/railties-7.1.3.4/lib/rails/application.rb:426:in `initialize!'
```

Do you think it is possible to disable UI loading during migrations?

**Comment by ka8725** [comment]
Hi @pyromaniac sorry for this issue. Can you please try to disable it by setting `ActualDbSchema.config[:ui_enabled] = false` in the initializer? [For refs](https://github.com/widefix/actual_db_schema?tab=readme-ov-file#2-using-initializer-1)

We will make a proper fix. Thanks for your feedback!

**Comment by ka8725** [comment]
Fixed in release 0.7.8.
